MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -1922,51 +1923,52 @@ The :code:`C.ydb_buffer_t` structure, which is the
-:code:`BufferT` for data, :code:`BufferTArray` for subscripts,
-:code:`KeyT` for keys where a key in turn consists of a variable or
-lock resource name and subscripts, as discussed in `Concepts`_, and
-:code:`VariadicPlist` for calling functions with a variable number of
-parameters.
+There are three structures for the inteface between YottaDB and Go:
+:code:`BufferT` for data, :code:`BufferTArray` for a list of
+subscripts or a set of variable or lock resource names, :code:`KeyT`
+for keys where a key in turn consists of a variable or lock resource
+name and subscripts, as discussed in `Concepts`_.
 
 .. code-block:: go
 
     type BufferT struct {
-        cbuft      *C.ydb_buffer_t // Point to C structure describing data
+        cbuft      *C.ydb_buffer_t // Pointer to C structure describing data
     }
 
     type BufferTArray struct {
-        elemsAlloc int             // Number of elements allocated in array
-	elemsUsed  int             // Number of elements in use (≤elemsAlloc)
-	cbuftarray *C.ydb_buffer_t // Pointer to array of C structures describing data
+        elemsAlloc uint            // Number of elements allocated in array
+	elemsUsed  uint            // Number of elements in use
+	cbuftarray *C.ydb_buffer_t // Pointer to start of array of C structures describing data
     }
     
     type KeyT struct {
 	Varnm      BufferT         // Pointer to variable name struct
         SubAry     BufferTArray    // Pointer to subscript struct
     }
 
-    type VariadicPlist struct {
-        vplist     *C.gparam_list  // Pointer to C structures
-    
 As these structures contain pointers to storage allocated by YottaDB,
 allowing a structure to go out of scope without first driving its
-“free” method introduces a storage leak.  Where possible, use the
-Golang :code:`defer` statement to automatically drive the appropriate
-free methods when these blocks go out of scope.
-
-As none of the fields in the structures described in `Go Data
-Structures & Type Definitions`_ is directly accessible (all fields
-start with lower case), use the methods associated with each structure
-to access and modify them. In the case of :code:`KeyT`, use methods of
-its component fields.
-
-Go Structure Access Methods
-===========================
+:code:`Free()` method introduces a storage leak.  Where possible, use
+the Golang :code:`defer` statement to automatically drive the
+appropriate free methods when these blocks go out of scope.
+
+For those fields in the structures described here that are not
+directly accessible (because they start with lower case letters),
+there are methods associated with their containing structures to
+access and modify them.
+
+Methods for each structure are classified as either `Go Access
+Methods`_ or `Go Simple API`_ methods. `Go Access Methods`_ are
+methods implemented in the Go wrapper for managing the structures for
+data interchange or `Go Access Methods`_ that wrap functionality
```

Comment:
Yes, the second one should be Go Simple API methods. Also, “methods” is not part of the hyperlink because the section header is “Go Simple API” which has methods and functions. Yes, it's a trifle inconsistent/awkward…